site/docs/mark/mark.md

```diff
@@ -62,9 +62,9 @@ For example, a bar chart has `mark` as a simple string `"bar"`.
 }
 ```
 
-A mark definition object can contains the following values.
+A mark definition object can contains the following properties and all properties of the mark config:
```

Comment:
Would it make sense to link to it or include all properties in the table?

Reply:
Good catch.  I'll add a link

Reply:
Ping me when you added it.

Reply:
Done